src/command.rs

```diff
@@ -333,6 +333,7 @@ pub fn password(input: &[u8]) -> IResult<&[u8], AString> {
 ///                CRLF is parsed by upper command parser.
 /// ```
 #[cfg(not(feature = "ext_sasl_ir"))]
+#[cfg_attr(docsrs, doc(cfg(feature = "ext_sasl_ir")))]
```

Comment:
There should be a `not` here. But as discussed, let's not try to do this now and resolve it properly in a follow-up.

Reply:
I would remove this in this PR because this annotation is simply wrong.